apps/lfx-one/src/app/modules/committees/components/committee-overview/committee-overview.component.html

```
@@ -2,311 +2,255 @@
+                severity="secondary"
+                tooltip="Edit chairs"
+                data-testid="committee-overview-edit-chairs-btn"
+                (onClick)="startEditChairs()"></lfx-button>
+            }
+          </div>
+          @if (membersLoading()) {
+            <div class="flex flex-col gap-1.5">
+              <p-skeleton width="100%" height="1.75rem" />
+              <p-skeleton width="100%" height="1.75rem" />
+            </div>
+          } @else if (chairs().length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)